src/alchemtest/tests/test_amber.py

```diff
@@ -17,22 +18,57 @@ def dataset(self, request):
         return super(TestAMBER, self).dataset(request)
 
 
-# difficult -- has sub dicts under 'complex'/'solvated' -> 'decharge', 'recharge', 'vdw'
-# class TestBACEexample(BaseDatasetTest):
-#     @pytest.fixture(scope="class",
-#                     params = [(load_bace_example, ('complex',), (12,)),
-#                               ])
-#     def dataset(self, request):
-#         return super().dataset(request)
+# The BACE example dataset does not conform to the standard API because it
+# contains sub dicts under 'complex'/'solvated' -> 'decharge', 'recharge',
+# 'vdw' instead of lists of files. In order to use the same testing base class,
+# we create a fake data set for "complex" and "solvated" separatedly and pass
+# these accessor functions to the class.
 
+# fake access to conform to expected API
+def _load_bace_example_complex():
+    dset = load_bace_example()
+    return Bunch(data=dset.data['complex'],
+                 DESCR="BACE example: complex")
 
-# class TestInvalidFiles(BaseDatasetTest):
-#     # can't get the list of list in data
-#     @pytest.fixture(scope="class",
-#                     params = [(load_invalidfiles, (slice(None),), (6,)),
-#                               ])
-#     def dataset(self, request):
-#         return super().dataset(request)
+def _load_bace_example_solvated():
+    dset = load_bace_example()
+    return Bunch(data=dset.data['solvated'],
+                 DESCR="BACE example: solvated")
```

Comment:
There might be a way to reduce code duplication but I didn't see anything clean right away so I opted for clear copy&paste for what hopefully remains the single exception.